PokeDemo/Models/Sprite.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Sprite.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 06/07/22.
+//
+
+import Foundation
+
+struct Sprite: Decodable {
+    var name: String?
+    var id: Int?
+    var sprites: SpriteDetail?
+}
```

Comment:
Remove optionals

Reply:
Covered in 37f42463fea0c21c150bf660477077e13b91ba3a